test/plugin/scenarios/shenyu-2.4.x-scenario/config/expectedData.yaml

```diff
@@ -0,0 +1,80 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+segmentItems:
+  - serviceName: shenyu-http
+    segmentSize: nq 0
+    segments:
+      - segmentId: not null
+        spans:
+          - operationName: GET:/order/findById
+            parentSpanId: -1
+            spanId: 0
+            spanLayer: Http
+            startTime: nq 0
+            endTime: nq 0
+            componentId: 14
+            isError: false
+            spanType: Entry
+            peer: ''
+            tags:
+              - {key: url, value: 'http://localhost:8189/order/findById'}
+              - {key: http.method, value: GET}
+            refs:
+              - {parentEndpoint: /order/findById, networkAddress: 'localhost:8189',
+                 refType: CrossProcess, parentSpanId: 0, parentTraceSegmentId: not null,
+                 parentServiceInstance: not null, parentService: 'Apache-ShenYu', traceId: not null}
+            skipAnalysis: 'false'
+  - serviceName: Apache-ShenYu
+    segmentSize: nq 0
+    segments:
+      - segmentId: not null
+        spans:
+          - operationName: /order/findById
+            parentSpanId: -1
+            spanId: 0
+            spanLayer: Http
+            startTime: nq 0
+            endTime: nq 0
+            componentId: 99
+            isError: false
+            spanType: Exit
+            peer: 'localhost:8189'
+            tags:
+              - {key: url, value: 'http://localhost:8189/order/findById'}
+              - {key: http.method, value: GET}
+              - {key: http.status_code, value: '200'}
+            refs:
+              - { parentEndpoint: /http/order/findById, networkAddress: '',
+                  refType: CrossThread, parentSpanId: 0, parentTraceSegmentId: not null,
+                  parentServiceInstance: not null, parentService: 'Apache-ShenYu', traceId: not null }
+            skipAnalysis: 'false'
+      - segmentId: not null
+        spans:
+          - operationName: /http/order/findById
```

Comment:
https://github.com/apache/skywalking-java/pull/147/files#r846691996 is about this one. This ia Shenyu(127) entry span, but it doesn't include a ref(cross process).

I think in `client http request -> ShenYu gateway（route to provider） -> shenyu-http(service provider)` chain,  `client http request -> ShenYu gateway` is not verified whether Shenyu's plugin could process the incoming call properly.

Reply:
Do you mean the following?
client http request ->  serviceA ->  gateway -> serviceB?

Reply:
Yes. Two services or one service have 2 URIs to cover the case and cost less.

Reply:
Thank you. I'll adjust it

Reply:
The test module has been adjusted.
![image](https://user-images.githubusercontent.com/12478263/162616341-ccbfa62f-199e-46de-b268-b79090e7209e.png)

Reply:
Local integration test passed.